EIPS/eip-6963.md

```diff
@@ -77,7 +77,7 @@ ipfs://QmZo7gsostaUdrV1peFz4cX6Z2TgriFJJP4VahG5knVm29
 https://ethereum.org/static/f541df14fca86543040c113725b5bd1a/99bcf/metamask.webp
 ```
 
-Additionally the image must be squared with 96x96px minimum resolution. Image format is recommended to be lossless like PNG and WebP or alternatively vectorial like SVG. We strongly discourage lossy formats like JPG/JPEG.
+The `icon` string MUST be a data URI as defined in [^RFC 2397]. The image SHOULD be squared with 96x96px minimum resolution. The image format is RECOMMENDED to be either lossless or vector based such as PNG, WebP or SVG to make the image easy to render on the DApp. Since SVG images can execute Javascript, applications and libraries MUST render SVG images using the `<img>` tag to ensure no untrusted Javascript execution can occur. The DApp SHOULD reject providers from being an available option with an icon using a lossy formats such as JPG or JPEG in order to prevent pixelation of images.
```

Comment:
Are we forbidding src URLs?

Rejecting based on lossy format seems a weird recommendation. While pixelation is mostly probably undesired, one could argue it could even be a purposeful artistic element.
And also we can't really say anything about image quality based only on its format.

Reply:
Yeah I agree. We should make it REQUIRED that a Data URI is used and also make it RECOMMENDED that SVG or lossless format is used

But rejecting a provider for using a lossy format like JPEG seems to extreme

Reply:
```suggestion
The `icon` string MUST be a data URI as defined in [^RFC 2397]. The image SHOULD be squared with 96x96px minimum resolution. The image format is RECOMMENDED to be either lossless or vector based such as PNG, WebP or SVG to make the image easy to render on the DApp. Since SVG images can execute Javascript, applications and libraries MUST render SVG images using the `<img>` tag to ensure no untrusted Javascript execution can occur. The DApp MAY reject providers from being an available option with an icon using a lossy formats such as JPG or JPEG in order to prevent pixelation of images.
```

Reply:
TBH I don't think any dapp would reject the provider for this reason. I don't think this would affect the functionality of the provider as well. Is there any other reason for not accepting JPG or JPEG format besides the pixelation issue?

Reply:
I'm thinking it's probably fine to accept and discourage, going to modify this text and link to commit.